checks/shell_download_validate.go

```diff
@@ -428,6 +429,12 @@ func isGoUnpinnedDownload(cmd []string) bool {
 		}
 
 		pkg := cmd[i+1]
+		// Consider strings that do not start with github.com
+		// as referring to a local path or an golang package
+		// from golang.org.
+		if !strings.HasPrefix(pkg, "github.com/") {
+			return false
+		}
```

Comment:
Why add this? If someone had the command `go install google.golang.org/protobuf/cmd/protoc-gen-go@latest`, then `isGoUnpinnedDownload` would return false when I think it should return true.

Reply:
thanks for catching it. I'd like to catch instances like https://github.com/envoyproxy/envoy/blob/main/examples/load-reporting-service/Dockerfile-lrs#L9, where the installation is not pinned but it's using a local folder. Any idea how I coould do this?

Reply:
You'd like scorecard to flag `go install /local/folder` as an unpinned dependency? If so, then I think you could just remove lines 435-437.

Reply:
no, the opposite: I'd like to *not* flag these as unpinned, since they are either:
1. committed to the repo, so they are pinned
2. downloaded from some random location, in which case other parts of our code will catch them (curl followed by install, etc). Note this is not implemented yet

Reply:
Ok, that sounds like a good thing to do. I'd probably recommend using a regex to detect if the arg passed to `go install` is in the format of a url something like `\w+\.\w+/\w+`. If it doesn't match that regex, assume it's a local download and return false.